plugins/plugin-webpack/plugin.js

```diff
@@ -67,7 +68,9 @@ function emitHTMLFiles({ doms, jsEntries, stats, baseUrl, buildDirectory }) {
 
         for (const jsFile of jsFiles) {
           const scriptEl = dom.window.document.createElement("script");
-          scriptEl.src = path.posix.join(baseUrl, jsFile);
+          scriptEl.src = baseUrl.startsWith('http')
```

Comment:
In other parts of the codebase, we've done `if (url.parse(resolvedImportUrl).protocol) {` which is slightly more general than just checking for `http`. Would love to see that here as well

Reply:
I had a look at how to implement `url.parse(resolvedImportUrl).protocol` to resolve the URL and as far as I can tell I have to import some stuff which needs some configuration where I have no clue how to get it inside the plugin context.

Mainly this part: https://github.com/pikapkg/snowpack/blob/master/snowpack/src/commands/dev.ts#L554.

I get the feeling that this would involve some bigger refactorings.

Reply:
I made the change that I was picturing, hopefully that's more clear / no big refactoring required at this point.